Exercise_1.py

```diff
@@ -1,22 +1,37 @@
+#Author: Henil Doshi
+
 class myStack:
      def __init__(self):
+          self.stack = []
          
      def isEmpty(self):
+          return len(self.stack) == 0
          
      def push(self, item):
+          self.stack.append(item)
          
      def pop(self):
-        
+          popElement = self.stack[len(self.stack)-1]
+          self.stack.pop()
+          return popElement
         
      def peek(self):
+          return self.stack[len(self.stack)-1]
```

Comment:
Please add edge cases for peek and pop to prevent from operating on an empty stack.

Reply:
Okay, thank you for the suggestions. I will make the required changes.